docs/source/tutorials/using_text.rst

```diff
@@ -0,0 +1,461 @@
+##########
+Using Text
+##########
+
+There are two different way by which you can render **Text** in videos:
+
+1. Using Pango (:mod:`~.text_mobject`)
+2. Using LaTeX (:mod:`~.tex_mobject`)
+
+If you want to render simple Text, you should use either :class:`~.Text` or 
+:class:`~.MarkupText` or one of it's detrivaties like :class:`~.Paragraph`.
+See :ref:`using-text-objects` for more information.
+
+LaTeX should only be used when you need Mathematical Typesetting. See 
+:ref:`rendering-with-latex` for more information.
+
+.. _using-text-objects:
+
+Text Without LaTeX
+******************
+
+The simplest way to add text to your animations is to use the :class:`~.Text`
+class. It uses the `Pango library`_ to render text. With Pango, you can also
+render non-English alphabets like `你好` or  `こんにちは` or `안녕하세요` or
+`مرحبا بالعالم`.
```

Comment:
Is it intended that the non-unicode text is set in literals? (Code literals would need a double pair of apostrophes.)

Reply:
I think that can be changed to
```suggestion
render non-English alphabets like 你好 or  こんにちは or 안녕하세요 or
مرحبا بالعالم.
```